lib/pg_ha_migrations.rb

```diff
@@ -75,6 +75,9 @@ def self.configure
 
   # Some methods require table / schema names to be in a specific format.
   InvalidIdentifierError = Class.new(StandardError)
+
+  # Some methods require the part_config entry to be in a specific state.
+  InvalidPartConfigError = Class.new(StandardError)
```

Comment:
Nit: I might consider renaming to `InvalidPartitionConfigError` or `InvalidPartmanConfigError` to be more clear if this error occurs.